test/net/sourceforge/kolmafia/DataFileMechanicsTest.java

```diff
@@ -30,7 +30,7 @@ public static Stream<Arguments> data() {
         Arguments.of("cafe_booze.txt", 1, 2, 2),
         Arguments.of("cafe_food.txt", 1, 2, 2),
         Arguments.of("classskills.txt", 6, 6, 7),
-        Arguments.of("coinmasters.txt", 2, 4, 5),
+        Arguments.of("coinmasters.txt", 3, 4, 7),
```

Comment:
this should probably be removed entirely, as the new format can have an unlimited number of currencies (right?)

Reply:
In theory, yes, although it looks like shop.php always has 5 costs in it (some of which are empty).
In which case the max should be 8, not 7. :(